README.md

```diff
@@ -1 +1,98 @@
+* Linux users (and adventurous MacOS users) can separately install:
+    * [Docker](https://docs.docker.com/engine/installation/linux/)
+    * [Docker Compose](https://docs.docker.com/compose/install/)
+    * [Docker Machine](https://docs.docker.com/machine/install-machine/) (for specifically running in a VM instead of locally)
+    * [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+
+## Quickstart
+
+### Vagrant
+By default, this will launch a Virtualbox VM running Ubuntu 14.04 LTS (a "docker host VM") and will run docker
+images within this VM.  Additionally, this VM will expose ports on your local machine (e.g. for viewing with a web
+browser).  This pattern works on all platforms (Linux, Mac OS, and Windows).  Linux hosts may choose to configure
+Vagrant to launch Docker on their own machine, rather than launching a separate VM.  This is discussed in a separate
+section.
+
+* `mvn install`
+    This is necessary because none of the apix-poc images or artifacts are published to
+    a repository for download; everything needs to be build from scratch locally.
+* `vagrant up`
+* try `vagrant up` again if something fails (e.g. failed download)
+
+For more information see [vagrant.md](vagrant.md)
+
+### Docker Compose
+On Linux machines with a docker engine running, Docker Compose can be used to run api-x docker images directly on
+the local machine.  Otherwise, docker may be run inside of a virtual machine.
+
+* For Mac, Windows, and Linux users wishing to run docker in a VM, start docker machine.  Linux users who
+  have Docker engine (> 1.10) on their local machine can skip this step
+    * `docker-machine create -d virtualbox --virtualbox-memory "2048" apix`
+    * `eval $(docker-machine env apix)`
+    * _Make sure that the api-x code (the fcrepo-api-x) directory is accessible to the docker machine VM_.  For
+    mac users, if the code is in your home sirectory somewhere under /Users, then you're good.  Linux users
+    have some work to do, see [setting up docker machine](docker/README.md)
+* run `mvn clean install` to build the api-x code
+* run `docker-compose up -d` to build & run docker images of API-X services.
+
+For more information see [docker/README.md](docker/README.md)
+
+## Verification
+These verification steps can be used to assure that the docker instance(s) are running and are accessible.
+
+First, determine the address `${ADDR}` of the machine running Docker
+* For Vagrant users, this is pre-configured to be `10.11.8.11`
+* For Docker Compose users:
+	* For Linux duning services directly on a local docker, this is `localhost`
+	* For Docker Machine users, you can find this by running `docker-machine ip apix`
+
+Next, verify that some services are running
+* Verify Fedora is running. Point your browser to `http://${ADDR}:8080/rest`
+	* You should see Fedora, with an empty root container
+	* Create an object through Fedora's UI
+* point your browser to Fuseki at `http://${ADDR}:3030/fcrepo`
```

Comment:
That might be better changed to "fedora_triple_index" or something like that. Or maybe the confusion is just inevitable.